### PR TITLE
Docs: Document that root cluster's can't populate OS users from leaves.

### LINKED ISSUE #14949: docs: mention that you must type the OS User to start an SSH session on a leaf cluster
*Update*: Teleport is working as intended here, but this behavior came as a surprise, so the task here is to update the trusted cluster documentation to clarify.

The root cluster's web UI can not populate the OS users for leaf cluster nodes, because it knows nothing about leaf cluster roles. As a result, users must manually type in the user rather than selecting it from a dropdown.

*Original issue content below..*

Expected behavior:
Ability for user to login to leaf cluster with listed user from the root cluster. The user in this case should be present as shown in screenshot. 
image (4)

Current behavior:
The expected user "Ubuntu" is not present when admin attempts to login to their leaf cluster. 

image (3)

Bug details:
- Teleport version
- V10.0.2
- Recreation steps
- Login to Root Cluster and attempt to authenticate to an existing Leaf Cluster
- Debug logs

gz#6010

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #14949 

Document that root cluster web UIs cannot populate OS users for leaf cluster hosts.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: docs/pages/management/admin/trustedclusters.mdx [hunk 1]
```diff
@@ -809,10 +809,12 @@ these pieces of data from it. It then performs the following actions:
   type="tip"
   title="Note"
 >
+Trusted Clusters work only in one direction. In the example above, users from
+the leaf cluster cannot see or connect to Nodes in the root cluster.
 
-  Trusted Clusters work only in one direction. In the example above, users from
-  the leaf cluster cannot see or connect to Nodes in the root cluster.
-
+The Teleport root cluster's web UI will not populate available OS users for
```
Comment (ptgott) [inline docs/pages/management/admin/trustedclusters.mdx:815]: Should we capitalize "web UI" throughout the docs? Right now the balance is: 

```
 $ grep -RhEoi "web UI" docs/pages | sort | uniq -c
 166 Web UI
  43 web UI
```

File: docs/pages/management/admin/trustedclusters.mdx [hunk 2]
```diff
@@ -809,10 +809,12 @@ these pieces of data from it. It then performs the following actions:
...
+leaf cluster nodes. When using the web UI, type in the host user when creating
```
Comment (ptgott) [inline docs/pages/management/admin/trustedclusters.mdx:816]:
```suggestion
leaf cluster Nodes. When using the web UI, type in the host user when creating
```
To use consistent capitalization in this paragraph

**Review by ptgott — APPROVED** [review]
Approved with minor comments

File: docs/pages/management/admin/trustedclusters.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/management/admin/trustedclusters.mdx:815]:
```suggestion
The Teleport root cluster's Web UI will not populate available OS users for
```

**Pull request merged**